### ISSUE TITLE
[Bug]: Error [ERR_REQUIRE_ESM]: require() of ES Module ...... not supported.

### ISSUE THREAD

**Issue by zolzaya** [vuestorefront/storefront-ui#2699]
### Describe the Bug

I'm using storefront-ui with Remix.run. But when I upgrade to `2.2.0` it shows me below errors

### Expected behavior

It should work.

### What package and version of StorefrontUI are you using?

@storefront-ui/react@2.2.0

### What framework does your application use?

React, Remix

### What version of Node.js are you using?

v19.8.1

### What browser (and version) are you using?

Chrome

### What operating system (and version) are you using?

macOS

### Relevant log output

```shell
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/zoloo/code/ecomm/storefront/node_modules/react-merge-refs/dist/index.mjs not supported.
Instead change the require of /Users/zoloo/code/ecomm/storefront/node_modules/react-merge-refs/dist/index.mjs to a dynamic import() which is available in all CommonJS modules.
    at Module.<anonymous> (/Users/zoloo/code/ecomm/storefront/node_modules/@storefront-ui/react/dist/hooks/useScrollable/useScrollable.cjs.js:1:108)
    at Module.<anonymous> (/Users/zoloo/code/ecomm/storefront/node_modules/@storefront-ui/react/dist/index.cjs.js:1:5663)
    at Object.<anonymous> (/Users/zoloo/code/ecomm/storefront/build/index.js:1028:21)
    at /Users/zoloo/code/ecomm/storefront/node_modules/@remix-run/serve/dist/index.js:43:17
    at Layer.handle [as handle_request] (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:144:13)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:140:7)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:140:7)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:140:7)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:140:7)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:140:7)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:140:7)
    at Route.dispatch (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/layer.js:95:5)
    at /Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:284:15
    at param (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:365:14)
    at param (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:376:14)
    at Function.process_params (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:421:3)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:280:10)
    at logger (/Users/zoloo/code/ecomm/storefront/node_modules/morgan/index.js:144:5)
    at Layer.handle [as handle_request] (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:328:13)
    at /Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/express/lib/router/index.js:280:10)
    at SendStream.error (/Users/zoloo/code/ecomm/storefront/node_modules/serve-static/index.js:121:7)
    at SendStream.emit (node:events:512:28)
    at SendStream.error (/Users/zoloo/code/ecomm/storefront/node_modules/send/index.js:270:17)
    at SendStream.onStatError (/Users/zoloo/code/ecomm/storefront/node_modules/send/index.js:417:12)
    at next (/Users/zoloo/code/ecomm/storefront/node_modules/send/index.js:759:28)
    at /Users/zoloo/code/ecomm/storefront/node_modules/send/index.js:767:23
    at FSReqCallback.oncomplete (node:fs:195:21)
```

### Able to fix / change the documentation?

- [ ] Yes
- [X] No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Szymon-dziewonski** [comment]
Hello @zolzaya, thank you for reaching out, seems like our sub dependency does not support CommonJs modules, we will look into it.

**Comment by zolzaya** [comment]
Okay thanks for info 👍

**Comment by Szymon-dziewonski** [comment]
@zolzaya we fixed issue on our side, will be fixed in another version

**Comment by FRSgit** [comment]
Hey @zolzaya - the fix just got released in [version 2.2.1](https://www.npmjs.com/package/@storefront-ui/react/v/2.2.1) 🎉

**Comment by zolzaya** [comment]
Thank it's working now
